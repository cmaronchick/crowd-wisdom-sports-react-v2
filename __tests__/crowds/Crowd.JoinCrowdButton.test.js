import React from 'react';
import JoinCrowdButton from '../../src/components/Crowd.JoinCrowdButton'
import renderer from 'react-test-renderer'

describe('it should create a button', () => {

})

test('present a button', () => {
    const button = renderer.create(<JoinCrowdButton />).toJSON()
    expect(button).toMatchSnapshot();
})