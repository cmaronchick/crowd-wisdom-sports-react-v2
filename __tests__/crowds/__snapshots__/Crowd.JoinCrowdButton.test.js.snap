// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`present a button 1`] = `
<button
  className="btn btn-primary"
  disabled={false}
  type="button"
>
  Join Crowd
</button>
`;
